Lesson_17/home_work/script.js

```diff
@@ -0,0 +1,680 @@
+		}
+		alert(result);
+	}
+// Task 2
+function calculateSum(){
+	var n = parseInt(prompt('', 'Enter the number to calculate the sum'));
+	var sum = 0;
+	for (var i = sum; i <= n; i++){
+		sum += i;
+	}
+	alert('The is ' + sum);
+}
+// Task 3
+function calculateMult(){
+	var n = parseInt(prompt('', 'Enter the number to multiple the sum'));
+	var sum = 1;
+	for (var i = sum; i <= n; i++){
+		sum *= i;
+	}
+	alert(sum);
+}
+// Task 4
+function calculateNumbers(){
+	var number = getNumber(),
+		plusVal = 0,
+		minusVal = 0,
+		zeroVal = 0,
+		message;
+	
+	function getNumber(){
+		var n = parseInt(prompt('', 'Enter some number'));
+		
+		if (n == 0){	
+			return true;
+		} else if (!isNaN(n)){
+			return n;
+		}
+		
+		return false;
+	}
+	
+	while (number){
+		
+		if (number === true){
+			zeroVal++;
+		} else if (number < 0){
+			minusVal++;
+		} else {
+			plusVal++;
+		}
+		
+		number = getNumber();
+	}
+	message = 'More than zero are ' + plusVal + ' values \n' + 'Less than zero are ' + minusVal + ' values \n' + 'Zeroes are ' + zeroVal + ' values';
+	alert(message);
+}
+// Task 5
+function transferValue(){
+	var values = [2.1366, 2.5, 0.3048, 3.7325, 29.86, 0.40951, 0.7112, 4.2657, 25.3995],
+		names = ['сажень', 'дюйм', 'фут', 'драхма', 'унция', 'фунт', 'аршин', 'золотник', 'дюйм'],
+		ourNames = ['м.', 'cм.', 'м.', 'г.', 'г.', 'кг.', 'м.', 'г.', 'мм.'],
+		all = '';
+		
+	function getFullName(name){
+		switch (name){
+			case 'м.': return 'метр';
+				break;
+			case 'cм.': return 'сантиметр';
+				break;
+			case 'г.': return 'грам';
+				break;
+			case 'кг.': return 'кілограм';
+				break;
+			case 'мм.': return 'міліметр';
+				break;
+		}
+	}
+	
+	for (var i = 0; i < values.length; i++){
+		
+		all += names[i].toUpperCase() + ' до ' + getFullName(ourNames[i]).toUpperCase() + ': \n';
+		
+		for (var j = 1; j < 11; j++){
+			console.log( j + ' ' + names[i] + ' = ' + j * values[i] + ' ' + ourNames[i] );
+			all +=  j + ' ' + names[i] + ' = ' + j * values[i] + ' ' + ourNames[i] + ' \n ';
+		}
+		
+		all += '\n';
+	}
+	alert(all);
+}
+// Task 6
+function getAcountValue(){
+	
+	var sum = parseInt(prompt('', 'Enter the sum of your money')),
+		years = parseInt(prompt('', 'Enter how many years you`d like to keep money on account'));
+	
+	for (var i = 0; i < years; i++){
+		sum *= 1.03;
+	}
+	sum = Math.round(sum);
+	alert(sum);
+	
+}
+// Task 7
+function getTheBiggest(){
+	
+	var numbers = prompt('', 'Enter ten pairs of numbres like in example: X, X; and then press "OK"'),
+		numberArr = numbers.split('; ');
+	
+	for (var i = 0; i < numberArr.length; i++){
+		var result = numberArr[i].split(', ');
+		
+		if (parseInt(result[0]) > parseInt(result[1])){
+			alert('The biggest number is ' + result[0]);
+		} else {
+			alert('The biggest number is ' + result[1]);
+		}
+		
+	}
+	
+}
+// Task 8
+function showDividedOn(){
+	var number = new Array;
+	
+	for (var i = 20; i < 51; i++){
+		if ((i % 3 === 0) && (i % 5 !== 0)){
+			number.push(' ' + i);
```

Comment:
this should be replaced with `number.push (i)` and below the body of the loop, in alert `number.join(' ')`